lib/beaker-puppet/helpers/puppet_helpers.rb

```diff
@@ -845,8 +858,15 @@ def sign_certificate_for(host = [])
                 fail_test("Failed to sign cert for #{hostnames}")
                 hostnames.clear
               end
-              on master, puppet("cert --sign --all --allow-dns-alt-names"), :acceptable_exit_codes => [0,24]
-              out = on(master, puppet("cert --list --all")).stdout
+
+              if version_is_less(puppet_version, '5.99')
+                on master, puppet("cert --sign --all --allow-dns-alt-names"), :acceptable_exit_codes => [0,24]
+                out = on(master, puppet("cert --list --all")).stdout
+              else
+                on master, 'puppetserver ca sign --all', :acceptable_exit_codes => [0, 24]
+                out = on(master, 'puppetserver ca list --all').stdout
+              end
+
               if hostnames.all? { |hostname| out =~ /\+ "?#{hostname}"?/ }
```

Comment:
Did we not need to change this after all?

Reply:
Oh right. Tests are running now. I'll let you know.

Reply:
Kinda looks like this test is just for efficiency. So it may "work" even if the check is failing.